tests/abi/tools/revng-check-compatibility-with-abi/ABIArtifactParser.cpp

```diff
@@ -65,7 +65,7 @@ struct llvm::yaml::MappingTraits<State::Register> {
   static void mapping(IO &IO, State::Register &N) {
     IO.mapRequired("Name", N.Name);
     IO.mapRequired("Value", N.Value);
-    IO.mapRequired("Bytes", N.Bytes);
+    IO.mapOptional("Bytes", N.Bytes);
```

Comment:
This change seems unrelated to the rest of the commit, but OK.

Reply:
It is related: because they are now allowed to just contain a "`Pointer`", they don't HAVE TO provide "`Bytes`" in all the cases.

Reply:
I guess I should have mentioned that in the commit message.

Reply:
I went the other way around after all. In the "pointer" cases, the "bytes of the pointer" are written there. That simplifies (or rather, generalizes) some other checks.

As such, this change is gone, so there's no need to comment on it. I'm closing the thread.